src/manifest.firefox.json

```diff
@@ -84,7 +84,10 @@
   ],
   "web_accessible_resources": [
     "pages/trackers-preview/index.html",
-    "pages/notifications/youtube.html"
+    "pages/notifications/youtube.html",
+    "pages/notifications/pause-assistant.html",
+    "pages/notifications/pause-resume.html",
+    "pages/notifications/pause-feedback.html"
```

Comment:
how about we put all those in a single page?

Reply:
It is JS logic in one place or configuration in three places. Question is what is more likely to break